falcon/api.py

```diff
@@ -856,10 +856,44 @@ def _get_body(self, resp, wsgi_file_wrapper=None):
                     iterable = wsgi_file_wrapper(stream,
                                                  self._STREAM_BLOCK_SIZE)
                 else:
-                    iterable = helpers.CloseableStreamIterator(stream, self._STREAM_BLOCK_SIZE)
+                    iterable = helpers.CloseableStreamIterator(
+                        stream, self._STREAM_BLOCK_SIZE)
             else:
                 iterable = stream
 
             return iterable, None
 
         return [], 0
+
+    # ------------------------------------------------------------------------
+    # Helpers that do not require self
+    # ------------------------------------------------------------------------
+
+    # QUESTION(csojinb): Find out if this should go here or in `api_helpers` or
```

Comment:
I wasn't sure if the stuff in `api_helpers.py` is meant to be "public" or not? I don't see any other static methods here, so perhaps it is intended to collect all non-state-required helpers, even if they aren't meant to be "public". If that is the case, I can move this there.